Broken test for FRED

https://travis-ci.org/pydata/pandas-datareader/jobs/148725878

```
======================================================================
FAIL: test_fred (pandas_datareader.tests.test_fred.TestFred)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/pydata/pandas-datareader/pandas_datareader/tests/test_fred.py", line 30, in test_fred
    self.assertEqual(int(received), 16440)
AssertionError: 16475 != 16440
-------------------- >> begin captured logging << --------------------
requests.packages.urllib3.connectionpool: INFO: Starting new HTTP connection (1): research.stlouisfed.org
requests.packages.urllib3.connectionpool: DEBUG: "GET /fred2/series/GDP/downloaddata/GDP.csv HTTP/1.1" 301 267
requests.packages.urllib3.connectionpool: INFO: Starting new HTTPS connection (1): fred.stlouisfed.org
requests.packages.urllib3.connectionpool: DEBUG: "GET /series/GDP/downloaddata/GDP.csv HTTP/1.1" 200 1603
--------------------- >> end captured logging << ---------------------
```

--- a/pandas_datareader/tests/test_fred.py
+++ b/pandas_datareader/tests/test_fred.py
@@ -23,11 +23,13 @@ class TestFred(tm.TestCase):
 
         received = web.DataReader("GDP", "fred", start, end)['GDP'].tail(1)[0]
 
-        # < 7/30/14 16535 was returned
+        # < 2014-07-30 16535 was returned
         # self.assertEqual(int(received), 16535)
-        # < 8/20/15 16502 was returned
+        # < 2015-08-20 16502 was returned
         # self.assertEqual(int(received), 16502)
-        self.assertEqual(int(received), 16440)
+        # < 2015-08-01 16440 was returned
+        # self.assertEqual(int(received), 16440)
+        self.assertEqual(int(received), 16475)
 
         with tm.assertRaises(RemoteDataError):
             web.DataReader("NON EXISTENT SERIES", 'fred', start, end)

--- a/pandas_datareader/tests/test_fred.py
+++ b/pandas_datareader/tests/test_fred.py
@@ -27,7 +27,7 @@ class TestFred(tm.TestCase):
         # self.assertEqual(int(received), 16535)
         # < 2015-08-20 16502 was returned
         # self.assertEqual(int(received), 16502)
-        # < 2015-08-01 16440 was returned
+        # < 2016-08-01 16440 was returned
         # self.assertEqual(int(received), 16440)
         self.assertEqual(int(received), 16475)
 

--- a/pandas_datareader/tests/test_fred.py
+++ b/pandas_datareader/tests/test_fred.py
@@ -19,9 +19,15 @@ class TestFred(tm.TestCase):
         # FRED.
 
         start = datetime(2010, 1, 1)
-        end = datetime(2013, 1, 27)
+        end = datetime(2013, 1, 1)
 
-        received = web.DataReader("GDP", "fred", start, end)['GDP'].tail(1)[0]
+        df = web.DataReader("GDP", "fred", start, end)
+        ts = df['GDP']
+        self.assertEqual(ts.index[0], pd.to_datetime("2010-01-01"))
+        self.assertEqual(ts.index[-1], pd.to_datetime("2013-01-01"))
+        self.assertEqual(ts.index.name, "DATE")
+        self.assertEqual(ts.name, "GDP")
+        received = ts.tail(1)[0]
 
         # < 2014-07-30 16535 was returned
         # self.assertEqual(int(received), 16535)

